### PR TITLE
SkullStripping part

### PR DESCRIPTION
**Pull request by anege**

### CODE REVIEW

**Comment by Remi-Gau** [comment]
Cool !

Thanks @anege !

I will have a look a bit later and give you some feedback on this!

**Review by marcobarilari — CHANGES_REQUESTED** [review]
Hi Ane, thank you very much. I just had a very quick look at it, I tempted to propose to keep this PR open so that we can fill in the missing part before merging it.

File: src/batches/setBatchCoregistration.m [hunk 1]
```diff
@@ -2,28 +2,25 @@
 
 function matlabbatch = setBatchCoregistration(matlabbatch, ref, src, other)
   %
-  % Short description of what the function does goes here.
+  % Set the batch for corregistering the source images into the reference image???
```
Comment (Remi-Gau) [inline src/batches/setBatchCoregistration.m:5]:
```suggestion
  % Set the batch for coregistering the source images into the reference image
```

File: src/batches/setBatchCoregistrationFmap.m [hunk 2]
```diff
@@ -2,20 +2,21 @@
 
 function matlabbatch = setBatchCoregistrationFmap(BIDS, opt, subID)
   %
-  % Short description of what the function does goes here.
+  % Set the batch for the coregistration of functional maps
```
Comment (Remi-Gau) [inline src/batches/setBatchCoregistrationFmap.m:5]:
```suggestion
  % Set the batch for the coregistration of field maps
```

File: src/batches/setBatchReslice.m [hunk 3]
```diff
@@ -1,29 +1,24 @@
 % (C) Copyright 2020 CPP BIDS SPM-pipeline developers
 
-function matlabbatch = setBatchReslice(matlabbatch, referenceImg, sourceImages)
+function matlabbatch = setBatchReslice(matlabbatch, ref, src)
```
Comment (Remi-Gau) [inline src/batches/setBatchReslice.m:3]:
```suggestion
function matlabbatch = setBatchReslice(matlabbatch, referenceImg, sourceImages)
```

[the remark this answers is not in the document]
Comment (Remi-Gau) [reply src/batches/setBatchReslice.m:3]: indeed

File: src/batches/setBatchReslice.m [hunk 4]
```diff
@@ -1,29 +1,24 @@
...
-  %                consectetur adipiscing elit. Ut congue nec est ac lacinia.
-  % :type argin1: type
-  % :param argin2: optional argument and its default value. And some of the
-  %               options can be shown in litteral like ``this`` or ``that``.
-  % :type argin2: string
-  % :param argin3: (dimension) optional argument
+  % :param matlabbatch: list of SPM batches
+  % :type matlabbatch: structure
+  % :param ref: Reference image
+  % :type ref: string
```
Comment (Remi-Gau) [inline src/batches/setBatchReslice.m:14]:
```suggestion
  % :param referenceImg: Reference image
  % :type referenceImg: string
```

File: src/batches/setBatchReslice.m [hunk 5]
```diff
@@ -1,29 +1,24 @@
...
+  % :param src: Source images
+  % :type src: cell
```
Comment (Remi-Gau) [inline src/batches/setBatchReslice.m:16]:
```suggestion
  % :param sourceImages: Source images
  % :type sourceImages: cell
```

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
Super late to review this.

Sorry @anege 

Couple of minor changes and things to fix. If you don't have time to look at it I can take over. :-)

**Comment by anege** [comment]
What do I have to do?
Since you already suggested some changes, I just approved those changes by clicking "commit sugestion"...and I received a lot of "run failed" mails...
________________________________
De: Remi Gau <notifications@github.com>
Enviado: domingo, 29 de noviembre de 2020 17:21
Para: cpp-lln-lab/CPP_SPM <CPP_SPM@noreply.github.com>
Cc: Ane Gurtubay Antolin <ane.gurtubay@uclouvain.be>; Mention <mention@noreply.github.com>
Asunto: Re: [cpp-lln-lab/CPP_SPM] SkullStripping part (#188)


@Remi-Gau requested changes on this pull request.

Super late to review this.

Sorry @anege<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fanege&data=04%7C01%7Cane.gurtubay%40uclouvain.be%7C38c8e668034a4354e2d608d89482c6fb%7C7ab090d4fa2e4ecfbc7c4127b4d582ec%7C0%7C0%7C637422636685712219%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=HWrkP0nRvARdXnXAsnVJiVxPBZMNzXpPawRoF4Yi%2Bow%3D&reserved=0>

Couple of minor changes and things to fix. If you don't have time to look at it I can take over. :-)

________________________________

In src/batches/setBatchCoregistration.m<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcpp-lln-lab%2FCPP_SPM%2Fpull%2F188%23discussion_r532230809&data=04%7C01%7Cane.gurtubay%40uclouvain.be%7C38c8e668034a4354e2d608d89482c6fb%7C7ab090d4fa2e4ecfbc7c4127b4d582ec%7C0%7C0%7C637422636685722214%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=nWQbdhohjUDlHj5ZIzhr%2FCDs4kd283B4109EwjxJX%2FY%3D&reserved=0>:

> @@ -2,28 +2,25 @@



 function matlabbatch = setBatchCoregistration(matlabbatch, ref, src, other)

   %

-  % Short description of what the function does goes here.

+  % Set the batch for corregistering the source images into the reference image???



⬇️ Suggested change

-  % Set the batch for corregistering the source images into the reference image???

+  % Set the batch for coregistering the source images into the reference image



________________________________

In src/batches/setBatchCoregistrationFmap.m<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcpp-lln-lab%2FCPP_SPM%2Fpull%2F188%23discussion_r532230919&data=04%7C01%7Cane.gurtubay%40uclouvain.be%7C38c8e668034a4354e2d608d89482c6fb%7C7ab090d4fa2e4ecfbc7c4127b4d582ec%7C0%7C0%7C637422636685732205%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=kUnxi2ZTdb2ZwCF%2BHSe7PsbAtQiXyeohncFDDXAXXbU%3D&reserved=0>:

> @@ -2,20 +2,21 @@



 function matlabbatch = setBatchCoregistrationFmap(BIDS, opt, subID)

   %

-  % Short description of what the function does goes here.

+  % Set the batch for the coregistration of functional maps



⬇️ Suggested change

-  % Set the batch for the coregistration of functional maps

+  % Set the batch for the coregistration of field maps



________________________________

In src/batches/setBatchReslice.m<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcpp-lln-lab%2FCPP_SPM%2Fpull%2F188%23discussion_r532231061&data=04%7C01%7Cane.gurtubay%40uclouvain.be%7C38c8e668034a4354e2d608d89482c6fb%7C7ab090d4fa2e4ecfbc7c4127b4d582ec%7C0%7C0%7C637422636685732205%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=2B9e0%2FaqD%2BYsIIqVNsyKqcNQqAQ0IOOPePU8O4BLVS0%3D&reserved=0>:

> @@ -1,29 +1,24 @@

 % (C) Copyright 2020 CPP BIDS SPM-pipeline developers



-function matlabbatch = setBatchReslice(matlabbatch, referenceImg, sourceImages)

+function matlabbatch = setBatchReslice(matlabbatch, ref, src)



⬇️ Suggested change

-function matlabbatch = setBatchReslice(matlabbatch, ref, src)

+function matlabbatch = setBatchReslice(matlabbatch, referenceImg, sourceImages)



________________________________

In src/batches/setBatchReslice.m<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcpp-lln-lab%2FCPP_SPM%2Fpull%2F188%23discussion_r532231135&data=04%7C01%7Cane.gurtubay%40uclouvain.be%7C38c8e668034a4354e2d608d89482c6fb%7C7ab090d4fa2e4ecfbc7c4127b4d582ec%7C0%7C0%7C637422636685742203%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=INV8iq1NQAnou%2FfolFQ2U4xsrXvLYOAZ0vyA%2FB9qNhk%3D&reserved=0>:

> +  % :param ref: Reference image

+  % :type ref: string



⬇️ Suggested change

-  % :param ref: Reference image

-  % :type ref: string

+  % :param referenceImg: Reference image

+  % :type referenceImg: string



________________________________

In src/batches/setBatchReslice.m<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcpp-lln-lab%2FCPP_SPM%2Fpull%2F188%23discussion_r532231165&data=04%7C01%7Cane.gurtubay%40uclouvain.be%7C38c8e668034a4354e2d608d89482c6fb%7C7ab090d4fa2e4ecfbc7c4127b4d582ec%7C0%7C0%7C637422636685742203%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=zPOvV4dsUgNshyX4qiTMtU7VWJbxYi6Zi8KUw56bMG0%3D&reserved=0>:

> +  % :param src: Source images

+  % :type src: cell



⬇️ Suggested change

-  % :param src: Source images

-  % :type src: cell

+  % :param sourceImages: Source images

+  % :type sourceImages: cell



________________________________

In src/batches/setBatchReslice.m<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcpp-lln-lab%2FCPP_SPM%2Fpull%2F188%23discussion_r532231079&data=04%7C01%7Cane.gurtubay%40uclouvain.be%7C38c8e668034a4354e2d608d89482c6fb%7C7ab090d4fa2e4ecfbc7c4127b4d582ec%7C0%7C0%7C637422636685752197%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=igVaZywFBdm7IJdisnL4PMEOYSRw%2BnxKq8%2BNZ3tyOfI%3D&reserved=0>:

> @@ -1,29 +1,24 @@

 % (C) Copyright 2020 CPP BIDS SPM-pipeline developers



-function matlabbatch = setBatchReslice(matlabbatch, referenceImg, sourceImages)

+function matlabbatch = setBatchReslice(matlabbatch, ref, src)



indeed

**Comment by marcobarilari** [comment]
@anege receiving so many emails telling you are failing is not helpfulf for self-esteem. As Dante says: "Don't care about them, just look and go on." 

Don't worry about them, we will fix all those fails the time we will merge the branch `doc` into `dev`

**Review by Remi-Gau — APPROVED** [review]
This looks good for merge to me. :-)

**Comment by Remi-Gau** [comment]
> What do I have to do? Since you already suggested some changes, I just approved those changes by clicking "commit sugestion"...and I received a lot of "run failed" mails...

Hey @anege 

Yeah I agree that some of those warning emails can be a bit scary. I will try to explain that tomorrow in the github lab tour.

**Pull request merged**